recipes/astrometry/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "astrometry" %}
+{% set version = "0.78" %}
+
+package:
+  name: {{ name|lower}}
+  version: {{ version }}
+
+source:
+  fn: {{name}}-{{ version }}.zip
+  url: https://github.com/dstndstn/astrometry.net/archive/{{ version }}.zip
+  sha256: 85480b9636757c89ef5ccc7aac557409ac2ab377eb6c6eca862b16cdd4faefd9
+
+build:
+  number: 0
+  skip: True  # [osx or win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - swig
+    - make
+  host:
+    - xorg-libx11  # [linux]
+    - xorg-libxau  # [linux]
+    - xorg-libxext  # [linux]
+    - xorg-libxrender  # [linux]
+    - xorg-xproto  # [linux]
+    - numpy
+    - zlib
+    - jpeg
+    - netpbm
+    - libpng
+    - cfitsio
+    - astropy
+    - bzip2
+    - cairo
+    - gsl
+  run:
+    - zlib
+    - jpeg
+    - netpbm
+    - libpng
+    - cfitsio
+    - astropy
+    - numpy
```

Comment:
I guess you use the numpy C API? If so this should use pin_compatible (see conda-forge.org/docs).

Reply:
It don't use C api, I think. But I have to check this. I think the program just calls numpy in python scripts.

Reply:
Just to be clear, the conda-forge will build multiple package versions with numpy pinned dependecies?

Reply:
If it only uses the Python API you probably don't need it as a `host` dependency. The pinning for numpy is for all 1.X.Y releases so there is no need for multiple builds but if there is ever a 2.0.0 release I expect there will be a migrator to rebuild everything that pins `numpy`.

Reply:
I checked and it uses C api for some functions.